.ci/pytorch/build.sh

```diff
@@ -333,7 +357,7 @@ else
     SITE_PACKAGES="$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')"
     mkdir -p "$CUSTOM_OP_BUILD"
     pushd "$CUSTOM_OP_BUILD"
-    cmake "$CUSTOM_OP_TEST" -DCMAKE_PREFIX_PATH="$SITE_PACKAGES/torch" -DPYTHON_EXECUTABLE="$(which python)" \
+    cmake "$CUSTOM_OP_TEST" -DCMAKE_PREFIX_PATH="$SITE_PACKAGES/libtorchsplit;$SITE_PACKAGES/torch;$SITE_PACKAGES" -DPYTHON_EXECUTABLE="$(which python)" \
```

Comment:
If we are not doing USE_SPLIT_BUILD here, just default path. Perhaps we should not pass ``$SITE_PACKAGES/libtorchsplit;`` in this case ?
We can do something like this:
```
SPLIT_PKG_PREFIX=""
if [[ "$USE_SPLIT_BUILD" == true ]]; then
SPLIT_PKG_PREFIX="$SITE_PACKAGES/libtorchsplit;"
fi
````
then include:
```
DCMAKE_PREFIX_PATH="$SPLIT_PKG_PREFIX$SITE_PACKAGES/torch;$SITE_PACKAGES" -DPYTHON_EXECUTABLE="$(which python)" 
```

Reply:
ohh this is fair. Albeit if we are in a situation where this matters, we likely messed up.

But it's better to bug out early :)